onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -5365,17 +5365,58 @@ def aten_slice_copy(
     raise NotImplementedError()
 
 
+@torch_op("aten::slice_scatter", trace_only=True)
 def aten_slice_scatter(
-    self: TensorType,
-    src: TensorType,
-    dim: int = 0,
+    self: TTensor,
+    src: TTensor,
+    dim: Optional[int] = 0,
```

Comment:
```suggestion
    dim: int = 0,
```

Looks like dim is not optional